docs_outdoornav_user_manual/web_user_interface/ui_overview.mdx

```diff
@@ -421,3 +412,203 @@ below.
+    Will start/stop recording video using one of the
+    UGV camera(s) to the /opt/onav/saved_files/media/... folder and can be retrieved
+    using a tool such as Filezilla or by navigating to the Files section in the hamburger menu.
+
+    Settings: Select which camera the recording will come from.
+
+-   <img src="/img/outdoornav_images/task_audio.png" width="30px"/> <strong>Start/Stop Audio Recording:</strong>
+    Will start/stop recording audio using one of the
+    UGV microphone(s) to the /opt/onav/saved_files/media/... folder and can be retrieved
+    using a tool such as Filezilla or by navigating to the Files section in the hamburger menu.
+
+    Settings: Select which microphone the recording will come from.
+
+-   <img src="/img/outdoornav_images/task_undock.png" width="30px"/> <strong>Undock UGV: </strong>
+    Will undock the UGV from the autocharge dock. Once
+    completed, the UGV can be sent on autonomous missions. It is
+    often recommended to place the undock task first in the list of Waypoints or as a start Mission Task;
+    that way, the UGV will automatically continue towards its next
+    Waypoint once undocked.
+
+    :::note
+
+    If the users places the Undock Task in the start Mission event for a Waypoint Mission the first Waypoint should be 
+    approximately 2-3 meters behind the UGVs docked position.
+
+    :::
+
+-   <img src="/img/outdoornav_images/task_wait.png" width="30px"/> <strong>Wait: </strong>
+    Will pause and wait for the specified number of
+    seconds at the end of the Waypoint.
+
+    Settings: Enter the amount of time to wait, in seconds.
+
+-   <img src="/img/outdoornav_images/task_custom.png" width="30px"/> <strong>New Custom Task: </strong>
+    Creates a new custom task that is defined by the user. 
+    <center>
+      <figure>
+        <img src="/img/outdoornav_images/custom_task_settings.png"/>
+        <figcaption>Custom Task Settings Dialog</figcaption>
+      </figure>
+    </center>
+    
+    - Task Name: Task name that will show up in the list of available tasks on the UI.
+    - Action Server Name: The namespace of the custom task action server.
+    - Float CSV: A list of comma seperated float values that consist of the numerical inputs to the custom task.
+    - String CSV: A list of comma seperated string values that consist of the semantic inputs to the custom task.
+
+    See the [Custom Tasks](../features/custom_tasks.mdx) section 
+    for details on how to develop custom tasks for your application.
+
+
+:::note
+
+If a Waypoint/Goalpoint has more than one task assigned to it, the icon will
+be replaced with a sheet of paper icon like so: <img src="/img/outdoornav_images/task_multi.png" width="30px"/>
+
+:::
+
+### Autonomous Docking
+
+<center>
+  <figure>
+    <img src="/img/outdoornav_images/ui_map_with_dock.png" />

```

Comment:
image missing local dock/undock buttons as well as the description of their use. i don't think we've explained them anywhere

Reply:
So this may be something we consider changing. We at present do explain the local dock/undock buttons but it's in a different view.
See here:
https://docs.clearpathrobotics.com/docs_outdoornav_user_manual/0.10.0/web_user_interface/ui_manual_mode

However if this isn't clear than maybe I move the dock/undock highlighting over to the Main overview picture?

Reply:
@jmastrangelo-cpr I've moved the information to the overview picture and explain it there now instead of the manual page. I haven't modified the picture yet as I'm assuming we'll want a new picture entirely with the assisted teleop details.